Jenkinsfile

```diff
@@ -40,5 +40,5 @@
 // I.e. for testing library changes
 //@Library(value="pipeline-lib@your_branch") _
 
-packageBuildingPipeline(['distros' : ['centos7', 'centos8', 'leap15']])
+packageBuildingPipeline(['distros' : ['el8', 'leap15']])
```

Comment:
Please use the [`packageBuildingPipelineDAOSTest()`](https://github.com/daos-stack/pipeline-lib/blob/master/vars/packageBuildingPipelineDAOSTest.groovy) function here so we have some per-branch regression/compatibility testing for this package.